deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -157,6 +157,7 @@ Resources:
           - Action:
             - 'ssm:Delete*'
             - 'ssm:Put*'
+            - 'ssm:AddTags*'
```

Comment:
Can see why this is needed. But, no idea how we didn't hit this problem before...

Reply:
that is what is so strange.

Reply:
and why it has taken all afternoon.

Reply:
I think SSM Tags are relatively new and probably just started getting passed down from CFN

Reply:
it did not report it until i deleted the test stack it said before that ssm can't update tags.  But what is weird is the prod one worked right away.